neasqc_wp61/models/quantum/circuit.py

```diff
@@ -59,9 +60,9 @@ def __init__(
         self.observables = observables
         if not all(x < self.n_qubits for x in self.observables.keys()): 
             raise ValueError('Qubit index out of range.')
-        self.output_probabilities = output_probabilities
         self.device = qml.device(
             device_name, wires=self.n_qubits, **kwargs)
+        self.output_probabilities = output_probabilities
         self.data_rescaling = data_rescaling
 
     @abstractmethod
```

Comment:
Sentences can't really start with "Will", we need a subject here. Like "they", "it", "the circuit" or something.